src/expressions/debug/StackTraceGenerator.ts

```diff
@@ -74,7 +101,8 @@ export type Location = {
 	offset: number;
 };
 
-export type SourceRange = {
+export declare type SourceRange = {
```

Comment:
Why the declare instead of importing this type where it's used?

Reply:
It was because something somewhere was renaming the `comment` field inconsistently. This is now fixed.